scripts/snowflake_generate_grant_all/ensure_ci.go

```diff
@@ -35,8 +35,12 @@ func ensureCI(names []string) error {
 	matrix := strategy["matrix"].(map[string]interface{})
 
 	modules := sets.NewStringSet().Add(names...)
-	for _, module := range matrix["module"].([]interface{}) {
-		modules.Add(module.(string))
+	if _, ok := matrix["module"].([]interface{}); ok {
+		for _, module := range matrix["module"].([]interface{}) {
+			modules.Add(module.(string))
+		}
+	} else {
+		modules.Add(matrix["module"].(string))
```

Comment:
This is the code change I made. Let me know if this is appropriate/idiomatic for Go and/or the program itself.

Reply:
I think the infra team back then was really into code generation so I feel like this not surprising. I think the idiomatic way of doing this parsing though would be to make a type for the github action yaml and call something like     `err = yaml.Unmarshal(yamlFile, &config)` instead of doing type casting. However, I am not sure it is worth the half an hour to do this.

Reply:
In short, I think this is fine and good 👍🏻